imod/msw/sprinkling.py

```diff
@@ -76,31 +75,42 @@ def _render(
         mf6_dis: StructuredDiscretization,
         mf6_well: Mf6Wel,
     ):
+        def ravel_per_subunit(array: xr.DataArray) -> np.ndarray:
+            # per defined well element, all subunits
+            array_out = array.to_numpy()[:, well_row, well_column].ravel()
```

Comment:
Please pass ``well_row`` and ``well_column`` to this function, like so:

```suggestion
        def ravel_per_subunit(array: xr.DataArray, well_row: np.ndarray, well_column: np.ndarray) -> np.ndarray:
            # per defined well element, all subunits
            array_out = array.to_numpy()[:, well_row, well_column].ravel()
```

I'd also prefer it if you put this helper function outside the class and add it above in this module. You can name it ``_ravel_per_subunit`` to show it is a semi-private helper function.